lir/config/experiment_strategies.py

```diff
@@ -73,35 +69,51 @@ def __init__(self) -> None:
     def data(self) -> DataStrategy:
         return parse_data_strategy(pop_field(self._config, 'data'), self._output_dir)
 
-    def primary_metric(self) -> Callable:
-        metric_name = pop_field(self._config, 'primary_metric')
-        return parse_metric(metric_name, self._output_dir, self._config.context)
-
-    def aggregations(self) -> Sequence[Aggregation]:
-        aggregation = pop_field(self._config, 'aggregation', required=False)
-        if not aggregation:
+    def output_list(self) -> Sequence[Aggregation]:
+        config: ContextAwareDict = pop_field(self._config, 'output', required=False)
+        if not config:
             return []
 
         results: list[Aggregation] = []
+        for item in config:
+            # Normalise configuration into (class_name, args)
+            if isinstance(item, str):
+                class_name, args = item, ContextAwareDict(config.context)
+            elif isinstance(item, Mapping):
+                class_name = pop_field(item, 'method')
+                args = ContextAwareDict(config.context, item)
```

Comment:
dit zou denk ik `item.context` moeten zijn, in plaats van `config.context`. Of beter nog, `args = item`, want waarom een nieuwe dict maken?